src/Shared/ErrorPage/Views/ErrorPage.cshtml

```diff
@@ -31,12 +31,11 @@
         @foreach (var errorDetail in Model.ErrorDetails)
         {
             <div class="titleerror">@errorDetail.Error.GetType().Name: @{ Output.Write(HtmlEncodeAndReplaceLineBreaks(errorDetail.Error.Message)); }</div>
-            @{
```

Comment:
This wasn't valid Razor syntax, it spit out errors generating the code. I think Razor is nice and decides to work around the error but it's not actually correct.

The `</div>` ended the markup block so you're back in the body of the `foreach` which is a **code** context, and `@{` isn't needed.

Reply:
Did it give an error when you compiled it?

Reply:
Yep. It throws an error but still generates the output.